src/main/java/org/folio/rest/impl/PermsAPI.java

```diff
@@ -382,15 +380,11 @@ public void deletePermsUsersById(String userid, Map<String, String> okapiHeaders
                                    Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     try {
       String tenantId = TenantTool.tenantId(okapiHeaders);
-      Criteria idCrit = new Criteria();
-      idCrit.addField(ID_FIELD);
-      idCrit.setOperation("=");
-      idCrit.setVal(userid);
-      PostgresClient pgClient = PostgresClient.getInstance(vertxContext.owner(),
-          tenantId);
+      Criterion idCrit = getIdCriterion(userid);
+      PostgresClient pgClient = PostgresClient.getInstance(vertxContext.owner(), tenantId);
       pgClient.startTx(connection -> {
         pgClient.get(TABLE_NAME_PERMSUSERS, PermissionUser.class,
```

Comment:
should pass connection as first parameter

Reply:
Good catch